src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -403,6 +403,7 @@ def load_arguments(self, _):
         c.argument('os_type', arg_type=get_enum_type(OS_TYPES), help="Set the OS type for the app to be created.")
         c.argument('app_insights_key', help="Instrumentation key of App Insights to be added.")
         c.argument('app_insights', help="Name of the existing App Insights project to be added to the Function app. Must be in the same resource group.")
+        c.argument('app_insights_disable', arg_type=get_three_state_flag(return_label=True), help="Disable creating an Application Insights for logging.")
```

Comment:
NIT: help text - Disable creating application insights resource during functionapp create. No logs will be available.